components/SpecialText.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-	<b-card>
+	<b-card class="bg-info">
 		<h3>{{ item.header }}</h3>
-		<div class="bg-light">
+		<div v-for="section in item.text" :key="section.type" class="bg-light">
 			<b-card-text>
-				{{ item.text }}
+				<i v-if="section.type == 'p'">{{ section.content }}</i>
```

Comment:
Miksi näin? Ehdotan, että jätät muutoksen riville 2, mutta reverttaat kaiken muun ja korjaat SpecialText-objektien rakenteen content.js:ään. Elementin suunniteltu rakennehan oli: `{ id: int, header: string, text: string }`
Ellei sitten ole tarvetta muuttaa SpecialTextin rakennetta datan vaatimusten mukaan?

Reply:
@markusmasalin ?